lib/devise/jwt/cookie.rb

```diff
@@ -14,8 +14,8 @@ module JWT
     module Cookie
       extend Dry::Configurable
 
-      setting :name, 'access_token'
-      setting :secure, true
+      setting :name, default: 'access_token'
+      setting :secure, default: true
       setting :domain
```

Comment:
Might as well add a default here as well, just incase its not used by other users.

Reply:
I have added to the readme that they can add their domain. The original one didn't set a domain default here, and what would we set it to 🤔

Reply:
Ah yeah, good question, are we sure we are ok without a default being set?
If so then we can ignore this, otherwise we could just default to `''`, WDYT?

Reply:
I personally wouldn't default it to anything as otherwise it would set it to `''` which could cause issues for them. I believe if it's not set it will use the current domain like what our implementation was doing before we set it based on the ENV.

Reply:
Ok, well as long as the warning doesn't show, all good from me.